autopilot/sagemaker_autopilot_neo4j_portfolio_churn.ipynb

```diff
@@ -0,0 +1,916 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    DB_URL = 'neo4j://ec2-3-91-14-235.compute-1.amazonaws.com:7687'
</pre><p>For any variables you want the customer to fill in, wrap in brackets. ie <code>variable = "&lt;your-value&gt;"</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Done for the password.  I'm did some work to anonymize the URL.  I'd like to leave the general form there as I think it's really helpful for users.  Hopefully that's ok.</p>

Reply:
<p>I think it's fine to leave whatever helper values you want, but applying the <code>&lt;&gt;</code> brackets so it's clear what customers need to replace versus leave. For instance, as a novice to this example, I'm not sure if I need to replace the "compute-1", the "7687", etc, or just the X's. If it is just the X's, I would find this helpful: <code>DB_URL = "neo4j://ec2-&lt;XXX-XXX-XXX-XXX&gt;.compute-1.amazonaws.com:7687"</code> </p>

Reply:
That sounds like a good idea.  Changed per your suggestion.